dotCMS/html/portlet/ext/contentlet/contentlet_versions_inc.jsp

```diff
@@ -115,8 +115,12 @@
 			 	<% if (!working && canEdit && !live) { %>
 			 		<a  href="javascript: selectVersion('<%= vinode %>');">
 			 	<%} %>
-			 	<img src="/contentAsset/image/<%=vinode %>/fileAsset/?byInode=1&filter=Thumbnail&thumbnail_h=125&thumbnail_w=125" style="width:150px;height:150px;border:1px solid silver;padding:3px;"></a>
 
+				<%if(ident.getAssetName().toLowerCase().equals("svg")){%>

```

Comment:
Can we trust in the asset name in order to identify something as a **svg** image?